docs/experiment/general/evaluation/local-evaluation.md

```diff
@@ -31,22 +24,42 @@ Local evaluation is just [evaluation](./implementation.md)--a function which tak
 
 ![Diagram of a local evaluation SDK.](../../../assets/images/experiment/local-evaluation.drawio.svg)
 
-The only non-local part of local evaluation is getting flag configurations from Amplitude Experiment, but this can happen at an interval, and flags can be cached in-memory on the server-side for zero latency access.
+The only non-local part of local evaluation is getting flag configurations from Amplitude Experiment, but this can happen at an interval, and flags can be cached in-memory in the SDK for zero latency access.
```

Comment:
```suggestion
Getting flag configurations from Amplitude Experiment happens on the server. You can configure this to happen at an interval, and Amplitude can cache flags in memory in the SDK for zero-latency access
```

@bgiori Trying to rewrite this in active voice. Is this still accurate? Mainly, is the flag retrieval interval user-configurable?

Reply:
Ah this sentence is a mess.

```suggestion
Local evaluation flag configurations are loaded from the server on startup and stored in memory for access prior to each evaluation.
```